Assets/_BForBoss/_Core/Scripts/Character/PlayerDashBehaviour.cs

```diff
@@ -6,6 +6,11 @@
 
 namespace BForBoss
 {
+    public interface IPlayerDashEvents
```

Comment:
This is part of the delegate pattern. 

Instead of injecting an action via init, you do
`_dashBehaviour.PlayerDashEventDelegate = this(parent)`

and then implement this interface. 
`PlayerMovementBehaviour :IPlayerDashEvents, IPlayerSlideEvents, IPlayerWallRunEvents`

Reply:
There were too many injecting of actions, and variables of actions being stored in different places, and it's getting confusing